### DISCUSSION TITLE
Account suspended, no explanation email, no answer to ticket situation

### DISCUSSION THREAD

**Discussion by raybenhas** [community/community#112451] [Other Feature Feedback, Questions, & Ideas]
Nearly two weeks since my GitHub account got suspended, without receiving any email explaining the cause of the suspension, neither getting the ticket I submitted #2653891 answered, I had several personal projects in my old one that I don't want to lose and ones that I am still working on and not getting a notice for two weeks is quite frustrating.

**Comment by AidenJamesYt** [comment]
Sometimes emails from services like GitHub might end up in your spam or junk folder, so i recommend you check that

**Comment by AidenJamesYt** [comment]
if anything, just stay patient but continue to follow up on the ticket regularly until you receive an answer

dont spam them tho
